meilisearch-http/src/index/updates.rs

```diff
@@ -271,6 +281,13 @@ impl Index {
             }
         }
 
+        if let Some(ref synonyms) = settings.synonyms {
+            match synonyms {
+                Some(synonyms) => builder.set_synonyms(synonyms.clone().into_iter().collect()),
+                _ => builder.reset_synonyms(),
```

Comment:
can you explicitly add a `None` clause instead of wildcard (and do the same for the one above please)

Reply:
done :+1: